packages/core/src/auth/sso/server.ts

```diff
@@ -0,0 +1,209 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import http from 'http'
+import { getLogger } from '../../shared/logger'
+import { ToolkitError } from '../../shared/errors'
+import { Socket } from 'net'
```

Comment:
nit: for cases where we know a module is definitely node-only, we should start using the `node:` prefix on imports.


```suggestion
import http from 'node:http'
import { getLogger } from '../../shared/logger'
import { ToolkitError } from '../../shared/errors'
import { Socket } from 'node:net'
```